packages/skpm/src/commands/publish.js

```diff
@@ -250,6 +257,7 @@ export default asyncCommand({
         repo
       )
       if (!upstreamPluginJSON.existingPlugin) {
+        spinner.stop()
```

Comment:
Stop `spinner` instance to stop it from overwriting `prompt()`. Restarted later again, by simply using the `print()` function.

Reply:
Let's check if `spinner` exist here as well

Reply:
Thanks, good idea! Should be there now: https://github.com/skpm/skpm/pull/209/commits/29a58b3b69c0bae16ed721924b71e66a263feae0